xml/System/DateTime.xml

```diff
@@ -7008,18 +7008,11 @@ An object that is equivalent to the date and time contained in `s`.
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The resolution of this property depends on the system timer.  
-  
-|System|Approximate Resolution|  
-|------------|----------------------------|  
-|Windows NT 3.5 and later|10 milliseconds|  
-|Windows 98|55 milliseconds|  
-  
- Starting with the .NET Framework version 2.0, the return value is a <xref:System.DateTime> whose <xref:System.DateTime.Kind%2A> property returns <xref:System.DateTimeKind?displayProperty=fullName>.  
-  
- An alternative to using <xref:System.DateTime.UtcNow%2A> is <xref:System.DateTimeOffset.UtcNow%2A?displayProperty=fullName>. While the former indicates that a date and time value is Coordinated Universal Time (UTC) by assigning <xref:System.DateTimeKind?displayProperty=fullName> to its <xref:System.DateTime.Kind%2A> property, the latter assigns the date and time value the UTC time's offset (equal to <xref:System.TimeSpan.Zero?displayProperty=fullName>).  
-  
-   
+ The resolution of this property depends on the system timer, which depends on the underlying operating system. It tends to be between 0.5 and 15 milliseconds.
+
+ Starting with the .NET Framework version 2.0, the return value is a <xref:System.DateTime> whose <xref:System.DateTime.Kind%2A> property returns <xref:System.DateTimeKind?displayProperty=fullName>.
```

Comment:
> the return value is a DateTime whose Kind property returns System.DateTimeKind. 

Unrelated to this change, but this looks like another migration issue similar to https://github.com/dotnet/docs/issues/1932: it should specify *which* `DateTimeKind` it returns, and it does [on MSDN](https://msdn.microsoft.com/en-us/library/system.datetime.utcnow(v=vs.110).aspx):

>  the return value is a DateTime whose Kind property returns DateTimeKind.Utc.

Should I open a new issue for this?

Reply:
I'll fix this one but please do and I'll follow up with the team. We keep finding new occurrences here and there.

Reply:
Ok, done: https://github.com/dotnet/docs/issues/3118.